cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/utils/CCEmbeddedBroker.java

```diff
@@ -38,7 +38,7 @@ public CCEmbeddedBroker(Map<Object, Object> config) {
 
     try {
       // Also validates the config
-      KafkaConfig kafkaConfig = KafkaConfig.apply(config);
+      KafkaConfig kafkaConfig = new KafkaConfig(config, true);
```

Comment:
you could also just use `.apply(config, true)`?

Reply:
I did that originally but I think that wasn't compatible with 2.8.